tests/unit/_internal/models/test_model.py

```diff
@@ -0,0 +1,164 @@
+import os
+from typing import TYPE_CHECKING
+from datetime import datetime
+from datetime import timezone
+
+import fs
+import pytest
+import fs.errors
+
+from bentoml import Tag
+from bentoml.exceptions import BentoMLException
+from bentoml._internal.models.model import Model
+from bentoml._internal.models.model import ModelInfo
+from bentoml._internal.configuration import BENTOML_VERSION
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+expected_yaml = (
```

Comment:
would it be easier to write this with a multi-line string?

Reply:
probably, I wrote this originally as a bunch of f-strings before I realized that I needed to use format